### DISCUSSION TITLE
URGENT: ssl cert showing revoked

### DISCUSSION THREAD

**Discussion by ssuess** [EasyEngine/easyengine#1627] [Support Requests] [answered]
I added a subdomain to a site this morning and then the site started getting an ssl revoked error. I removed the alias but still showing revoked. Even after several force refreshes of my cert it is showing as revoked, I also refreshed the proxy but no dice. I’m losing my mind trying to fix this, can anyone help?

**Comment by ssuess** [comment, marked as answer]
I fixed it finally. After disabling ssl for the site,  I had to go into `/opt/easyengine/services/nginx-proxy/acme-conf/certs` AND `/opt/easyengine/services/nginx-proxy/certs` and delete all certs related to my site. Then when I re-added SSL it issued VALID new certs. What a nightmare.

**Reply by Th0rHere** [reply]
Can you tell me how you disabled ssl on the site 

Trying to do something similar

**Reply by ssuess** [reply]
ee site update sitename —ssl=“off”

**Reply by ssuess** [reply]
btw, the above command requires the latest version 4.4.0 of ee, it will not work on earlier versions.

**Reply by Th0rHere** [reply]
Just disabled it and then deleted the certs related to my site, but trying to add SSL again with wildcard and it still things I already have and throws "Update from normal SSL to wildcard SSL is not supported yet."

**Reply by ssuess** [reply]
Are you sure you deleted all the certs associated with your site in all the folders? I found 6 files I needed to delete, 3 in each of the folders above.

**Reply by Th0rHere** [reply]
Yeah, missed one of them lol, but trying to enable ssl is still throwing error trying set up ssl

Trying to to just start again and get site setup correctly with ssl and alias

**Comment by ssuess** [comment]
Sadly, this same thing happened to me again this morning, here are the details:
- I had two sites: example.com and new.example.com
- I deleted example.com (`ee site delete example.com`)
- I added example.com as an alias domain to new.example.com (`ee site update new.example.com --add-alias-domains="example.com"`)
- I then did an `ee shell` command `wp search replace "https://new.example.com" "https://example.com" --all-tables` (followed by `wp cache flush`)
Site was then broken, as before, with an error about the cert being revoked. so then I tried 
- `ee site update new.example.com --ssl="off"` followed by `ee site update new.example.com --ssl=le` which did not solve the problem
- I then ran `ee site update new.example.com --ssl="off"` one more time to turn it off
- Then (as before above) I went into the `/opt/easyengine/services/nginx-proxy/certs` folder and deleted the certs for new.example.com
- and finally ran `ee site update new.example.com --ssl=le` which did renew the certs and make both domains work.

**Comment by mrrobot47** [comment]
This has been fixed in `EE v4.5.0`. Please update to the latest version via `ee cli update`

**Reply by GreenLondon** [reply]
This is not fixed EE 4.6.1

ee site update new.example.com --ssl=off does not delete the certificates and the site throughs a 502 error

**Reply by swissbuechi** [reply]
Same issue on 4.6.2
